extensions/cpp/syntaxes/cuda-cpp.tmLanguage.json

```diff
@@ -0,0 +1,19991 @@
+{
+  "information_for_contributors": [
+    "This code was auto generated by a much-more-readable ruby file",
+    "This file essentially an updated/improved fork of the atom syntax",
+    "see https://github.com/jeff-hykin/better-cpp-syntax/blob/master"
+  ],
+  "version": "",
```

Comment:
All VS Code grammars are pulled from other repositories with no hand modification (there may be some some modification doen by a script). It looks like this one comes from jeff-hykin/better-cpp-syntax, is that correct? And is it correct to say that there is no hand modification done on this file?
To facilitate updating grammars from their upstream repositories, we have scripts. An update to the script will be needed somewhere around here: https://github.com/pooyazv/vscode/blob/c3b556727e2bc2472541c2c063a7194794ff09db/extensions/cpp/build/update-grammars.js#L11-L12

Reply:
I don't see the cuda grammar in jeff-hykin/better-cpp-syntax yet. If you can point me to where the grammar is I can make the script update.

Reply:
Thank you @alexr00. Regarding the following:
- add a test file and to make sure the new grammar is included in our scripts
are you referring to the test file we are adding for the colorizer test or is this a separate file we need to add.

Thank you

Reply:
@PooyaZv, I'm referring to the colorizer test file that I described here: https://github.com/microsoft/vscode/pull/119444#discussion_r598600634

Reply:
Great. Thank you. I have added a file to the colorizer tests. Thanks.

Reply:
Thank you @alexr00 and sorry for the delay. I have updated update-grammars.js to pull cuda--cpp.tmLanguage.json from our repo at [https://github.com/NVIDIA/cpp-cuda-grammar](https://github.com/NVIDIA/cpp-cuda-grammar) Just in case, Bob Brown is privy to more details. I would be happy to provide them over email as well. Thank you.